JSON_Bourne/script.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function() {
+	$.getJSON("http://localhost:60000/", function(obj) {
+	    document.getElementById("inst_name").innerHTML = "DEMO"
+		document.getElementById("config_name").innerHTML = "Configuration: " + obj.config_name
+
+		var group_titles = Object.keys(obj.groups)
+		for (i = 0; i < group_titles.length; i++) {
+			var title = group_titles[i]
+			var block_titles = Object.keys(obj.groups[title])
+			document.getElementById("groups").innerHTML += "<h3>" + title + "</h3>"
```

Comment:
Ideally I'd prefer the HTML to be created directly as DOM elements:

http://www.w3schools.com/jsref/met_document_createelement.asp

Creating the HTML code manually as a string is ugly and error prone